starter-code/js/tenSecondsMath.js

```diff
@@ -1,13 +1,187 @@
+  var overlay = createOverlay();
+  document.getElementById('parentOfCard').appendChild(overlay);
+  var counter;
+  var temporary = createQuestion(gameOption);
+  var question = temporary.question; // Create question
+  var answer = temporary.answer; // Create answer for comparison reasons
+  var questionDiv = document.createElement('div');
+  questionDiv.style.id = 'question';
+  questionDiv.innerText = String(question);
+  // console.log(questionDiv); JUST FOR TEST
+  var inputAnswer = createInputField();
+  document.getElementById('overlay').appendChild(questionDiv);
+  document.getElementById('overlay').appendChild(inputAnswer);
+
+// If a correct answer is entered and Enter button pressed, make sound, and add 10 seconds to timer. Then arrange for new question.
+  inputAnswer.onkeypress = function (e) {
+    if (e.keyCode === 13) {
+        if (parseInt(answer) === parseInt(inputAnswer.value)) {
+          var audio = new Audio('./audios/cash-register.mp3');
+          audio.play();
+          // console.log("success"); // JUST TEST
+          counter += 10;
+          var tempHolder = createQuestion(gameOption);
+          question = tempHolder.question;
+          answer = tempHolder.answer;
+          // console.log(question); // JUST TEST
+          questionDiv.innerText = String(question);
+          inputAnswer.value = null;
+        }
+      }
+
+  };
+  var timerDisplay = createTimerDisplay();
+  var timerLabel = document.createElement('div');
+  timerLabel.innerText = "seconds remaining";
+  timerLabel.setAttribute('id','timerLabel');
+  document.getElementById('overlay').appendChild(timerDisplay);
+  document.getElementById('theTimer').innerHTML = 10;
+  document.getElementById('overlay').appendChild(timerLabel);
+
+  var btn = document.getElementById("btnStart");
+  btn.disabled = true;
+  counter = 10;
+  var callbackFunction = function () {
+    console.log(counter);
+    timeoutId = setTimeout(callbackFunction, 1000);
+    counter -= 1;
+    document.getElementById('theTimer').innerHTML = counter;
 
+// If timer counter reaches zero, then clear timer and gameover.
+    if (counter === 0) {
+      clearTimeout(timeoutId);
+      btn.disabled = false;
+      removeElements(inputAnswer,questionDiv,timerLabel);
+      document.getElementById('theTimer').innerHTML = "<img width=250px src=\"./images/gameover.jpg\" alt=\"\">";
+      var btnRestart = createBtnRestart();
+      document.getElementById('overlay').appendChild(btnRestart);
+      console.log(document.getElementById('theTimer').innerHTML);
 
-// Returns an object with {question, answer}
+// Set event listening on restart button and remove the overlay if clicked to allow a new game.
+      var restartGameButton = document.getElementById('btnRestart');
+      restartGameButton.onclick = function (e) {
+        overlay.remove(overlay);
+      };
+    }
+  };
 
+  var timeoutId = setTimeout(callbackFunction, 1000);
 
-// Checks a user answer
+}
```

Comment:
Good job! If you have the time, you could try to change the radio buttons as you said, and also try to apply the conditions listed on the exercise for the operations:
The result should never be negative
The difference between the generated operators can't be less than 3 (except for the division)
The generated numbers should never be equal (except for the division)